manifests/quick-start-minimal.yaml

```diff
@@ -1669,6 +1761,14 @@ spec:
               - mkdir
               - -p
               - /data/my-bucket
+              - '&&'
+              - mkdir
+              - -p
+              - /data/my-bucket-2
+              - '&&'
+              - mkdir
+              - -p
+              - /data/my-bucket-3
         livenessProbe:
```

Comment:
wanted to be able to test with multiple buckets to make sure we're writing to and deleting from the right ones